src/modules/pmc/src/lib/Pmc.h

```diff
@@ -2,10 +2,13 @@
 // Licensed under the MIT License.
 
 #include <PmcBase.h>
+#include <string>
 
 class Pmc : public PmcBase
 {
 public:
     Pmc(unsigned int maxPayloadSizeBytes);
     ~Pmc() = default;
+
+    int RunCommand(const char* command, bool replaceEol, std::string* textResult, unsigned int timeoutSeconds) override;
```

Comment:
>, unsigned int timeoutSeconds) [](http://example.com/codeflow?start=80&length=30)

This module won't need timeouts either other than the default 60 seconds that is automatically applied when timeout to commonutils!ExecuteCommand is 0.

Reply:
For the `g_commandExecuteUpdate` we specify a timeout of 600 seconds. Could we keep it?

Reply:
Is it really necessary to have a timeout so long? This is 10 minutes. Default (when specified with a 0 value) is 60 seconds aka 1 minute.

If you need 10 minutes OK, but then recommend to hard-code that to some constant and remove this timeoutSeconds argument from this internal method.